README.md

```diff
@@ -177,8 +177,8 @@ See the example below
 ## Handling NGINX configuration and certificates
 
 ```yaml
- - name: "Sync NGINX configuration- multi file and certificate to NGINX for Azure"
-        uses: nginxinc/nginx-for-azure-deploy-action@v0.3.0
+ - name: "Sync NGINX configuration- multi file and certificate to NGINXaaS for Azure"
+        uses: nginxinc/nginxaas-for-azure-deploy-action@v0.3.0
```

Comment:
We should avoid renaming the repo or any references to it for now. We can take that on separately if the current repo name becomes a source of confusion for users.